### DISCUSSION TITLE
PhD student renewing my student package again; rejected 10 times

### DISCUSSION THREAD

**Discussion by sjohnsonyu** [community/community#58584] [GitHub Education]
### Hi everyone,

Hi there, I'm a PhD student at Harvard University (under the Graduate School of Arts and Sciences). I've renewed my student developer package several times throughout my undergrad, master's, and now PhD (the last one being Jan 18, 2023). I recently received an email saying I need to update within 30 days. This time, however, I've been having troubles with the renewal. 

Here are the things that I've tried/done:
- Updated my bio (including my middle name, which appears on official documentation)
- Taken a picture of my unofficial transcript (does not contain anticipated graduation date) --> says there's a problem with the date not demonstrating an affiliation
- Taken a picture of my Harvard University Advising Report (contains my grades and anticipated graduation date) --> says it doesn't include my school name, even though it says "Harvard University Advising Report" on top
- Taken a picture of my unofficial transcript and my student ID --> unable to verify but doesn't say why
- Taken a picture of my unofficial transcript, my Harvard University Advising Report, and my student ID --> says there is a problem with my bio (though it's been updated) and that there is no date demonstrating affiliation (but this is present in the advising report)
- Tried directly uploading JPG versions of PDFs of Advising Report and Unofficial Transcript --> won't accept PDFs

I've spent several hours on this today, which is kind of hard to believe, given how smoothly the renewal has gone in the past. Could you please help me?

**Comment by samus-aran** [comment]
Hi @sjohnsonyu 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process. I would also highlight that if names are switched around, or have dashes that these will be detected and used against your info.

Hope this helps 👍
